test/pages/index.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Home() {
```

Comment:
Playwright pings your base URL to know that your webServer is running. So I created this page.